chunks.go

```diff
@@ -61,9 +61,11 @@ type ChunkWriter interface {
 type chunkWriter struct {
 	dirFile *os.File
 	files   []*os.File
-	wbuf    *bufio.Writer
-	n       int64
-	crc32   hash.Hash
+	// XXX: In wal.go, we call the equivalent fields "cur" and "curN".
+	//      Should we make the naming consistent between both?
```

Comment:
Yea, more naming consistency cannot hurt.
In #61 I named them `fbuf` and `pos` – those might be the most expressive ones.